dashboards-observability/public/components/synthetics/home_view.tsx

```diff
@@ -96,28 +209,67 @@ export const SyntheticHomeTab = (props: LogHistoryProps) => {
       if (pplFilterValue === '') {
         setFilterVal('')
       } else {
-        setFilterVal(" | " + pplFilterValue);
+        setFilterVal(pplFilterValue);
       }
-      pplServiceRequestor(pieQuery + filterVal + timeVal + endPieQuery, "data", setPieChartLogs)
-      pplServiceRequestor(historyQuery + filterVal + timeVal, "jsonData", setHistoryLogs)
+      updateQueryComponents();
     }
   };
 
-  const showHelpFlyout = () => {
-    setAddVizDisabled(true);
-    setHelpIsFlyoutVisible(true);
-  };
 
-  const closeHelpFlyout = () => {
-    setAddVizDisabled(false);
-    setHelpIsFlyoutVisible(false);
-  };
 
-  let helpFlyout;
-  if (isHelpFlyoutVisible) {
-    helpFlyout = <PPLReferenceFlyout module="panels" closeFlyout={closeHelpFlyout} />;
+
+  const updateQueryComponents = async () => {
+    let finalPieQuery = queryAccumulator(pieQuery() + endPieQuery(), START_TIME_FIELD, start, end, filterVal); 
```

Comment:
This can be a const

Reply:
I tried including it in a const and it left the flyout permanently open. The way the flyouts were done is similar to how other components had flyouts, should I still change this somehow?

Reply:
Nah, it's fine!